src/styles/mixins/buttons.scss

```diff
@@ -1,10 +1,66 @@
-@import './focus';
+@use "./focus";
+@use "../variables" as var;
 
 @mixin reset-native-btn-styles {
-  @include outline-on-keyboard-focus;
+  @include focus.outline-on-keyboard-focus;
 
   padding: 0px;
   margin: 0px;
   background-color: transparent;
   border-style: none;
 }
+
+/**
+ * A <button> composed of an SVG icon (left) and text (right) with some
+ * hover transition effects
+ */
+@mixin action-button {
+  @include reset-native-btn-styles;
```

Comment:
question about our blue focus ring... Are we intending to only show this for keyboard navigation ? I may need to keep that in mind going forward

Reply:
...I don't know!

Reply:
I'll msg Jared on slack about this, I want get to an understanding about focus states, since I'm super confused :)

Reply:
> Are we intending to only show this for keyboard navigation ?

The "only show a focus ring on keyboard interaction" behavior was something I introduced as an improvement to "always hide the focus ring" approach which we were using previously for certain elements but which causes problems for users relying on keyboard navigation.

I'd suggest that whether to disable any default focus ring for touch/mouse input is context-specific, but we should make sure that any UI we build is keyboard-navigable.